cloud/passportauth/src/auth/DefaultStrategy.ts

```diff
@@ -0,0 +1,31 @@
+import { BunyanLogger, Logger } from '@raincatcher/logger';
+import { UserSecurityService } from '../user/UserSecurityService';
+
+const log: Logger = new BunyanLogger({ name: 'Passport-Auth', level: 'error' });
+import { User } from '../user/User';
+
+/**
+ * Default strategy to be used by Passport's local strategy. If user credentials are valid, proceed to login,
+ * otherwise, reject it. Calls serialize user with user login id.
+ *
+ * @param userSec {UserSec}
+ * @returns {Function} - Returns the default strategy function to be used by passport
```

Comment:
we should avoid using types in jsdocs, it's an extra point of maintenance. I think the doc generator we're using either ignores them in favor of the types given by the compiler or includes these as extra plaintext (which would be even worse).